style.css

```diff
@@ -0,0 +1,186 @@
+}
+
+h1 {
+    color: #373C99;
+}
+
+.landingTopContainer {
+    display: flex;
+    justify-content: space-between;
+}
+
+.landingLeft {
+    width: 20%;
+}
+
+.landingTitleContainer {
+    justify-content: space-between;
+}
+
+.leaderSurfboardContainer {
+    display: flex;
+}
+
+.mainContainer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+#mainGameContainer {
+    margin: 5%;
+    width: 1000px;
+    height: 600px;
+    position: relative;
+    border-radius: 10px;
+    background: linear-gradient(#f9855d,#fdd288,#ffffff);
+    overflow: hidden;
+}
+
+.waves {
+    position: relative;
+    width: 100%;
+    height: 30vh;
+    margin-bottom:-7px;
+    min-height:100px;
+    max-height:150px;
+    z-index: -2;
+    
+}
+
+.sun {
+    width: 100px;
+    height: 100px;
+    position: absolute;
+    top: 370px;
+    right: 40px;
+    background-color: #f9cb5d;
+    border-radius: 50%;
+    z-index: 2;
+}
+
+.ocean {
+    height: 150px; /* change the height of the waves here */
+    width: 400%;
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    overflow-x: hidden;
+    background-color: #1e90ff;
+    border-top: 2px #ffffff solid;
+    z-index: 4;
+}
+
+#avatar {
+    width: 60px;
+    height: 40px;
+    position: absolute;
+    left: 0px;
+    top: 415px;
+    z-index: 5;
+    background-image: url(./imgs/surfingPenguin.png);
+    background-size: cover;
+}
+
+#displaySplashInstructionsContainer {
+    position: absolute;
+    display: flex;
+    top: 0px;
+    left: 0px;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 600px;
+    width: 1000px; 
+    padding: 40px;
+    background-color: #FE9D8D;
+    border: 5px #373C99 solid;
+    z-index: 10;
+    /* display: none; */
+    /*comment this out as needed */
+}
+
+#leaderSurfboard {
+    border-radius: 10px;
+    background-color: #373C99;
+    color: #F0EFF5;
+    width: 150px;
+    margin-right: 50px;
+}
+
+.startGameButtonContainer {
+    display: flex;
+    justify-content: center;
+}
+
+#startGameButton {
+    border-radius: 10px;
+    background-color: #373C99;
+    color: #F0EFF5;
+    font-size: 1.5rem;
+    width: 150px;
+    height: 60px;
+}
+
+.wave {
+    background: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 800 88.7'%3E%3Cpath d='M800 56.9c-155.5 0-204.9-50-405.5-49.9-200 0-250 49.9-394.5 49.9v31.8h800v-.2-31.6z' fill='%23003F7C'/%3E%3C/svg%3E");
+    position: absolute;
+    width: 200%;
+    height: 100%;
+    animation: wave 20s -3s linear infinite;
+    transform: translate3d(0, 0, 0);
+    opacity: 0.6;
+  }
+  
+  .wave:nth-of-type(2) {
+    bottom: 0;
+    animation: wave 36s linear reverse infinite;
+    opacity: 0.7;
+  }
+  
+  .wave:nth-of-type(3) {
+    bottom: 0;
+    animation: wave 40s -1s linear infinite;
+    opacity: 0.5;
+  }
+
+.cloud-background {
+    position: relative;
+    overflow: hidden;
```

Comment:
You should be able to bin this overflow rule, since it's parent already has overflow hidden.

Reply:
Removed as requested.